modules/yarn.lock

```diff
@@ -3003,12 +2995,13 @@ babel-plugin-add-module-metadata@2.24.1:
     liferay-npm-build-tools-common "2.24.1"
     read-json-sync "^2.0.1"
 
-babel-plugin-alias-modules@2.23.0:
-  version "2.23.0"
-  resolved "https://registry.yarnpkg.com/babel-plugin-alias-modules/-/babel-plugin-alias-modules-2.23.0.tgz#f04c39c1da080b77f580baafb32d99bad784dd03"
-  integrity sha512-H+aFZWsgxuNChIlo/oeeRO3W0Bg6k1z3G5qWzjQoYtr3GsS8fFpFWMsur9r3YXbPJerVGqR2BouH44WKM60W5A==
+babel-plugin-add-module-metadata@2.24.2:
```

Comment:
A lot of these updates are suspicious, because now we have `babel-plugin-add-module-metadata` v2.24.1 _and v2.24.2 (and same duplication for other modules). To be clear, there was duplication before this PR too (eg. v2.23.0 + 2.24.1), which wasn't good either.

I think we need to de-dupe as described [at the bottom of CONTRIBUTING.md](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/npm-tools/CONTRIBUTING.md#integrating-in-liferay-portal), and if that doesn't work, dig in with `yarn why` to figure out why the duplicate packages are here.